### PR TITLE
Add DailyFeed by @paulsumit1993

### PR DESCRIPTION
**Pull request by paulsumit1993**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -11623,6 +11623,17 @@
       "license": "mit",
       "date_added": "Jan 25 2017",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "DailyFeed",
+      "source": "https://github.com/paulsumit1993/DailyFeedr",
```
Comment (dkhamsing) [inline contents.json]: should https://github.com/paulsumit1993/DailyFeed

File: contents.json [hunk 2]
```diff
@@ -11623,6 +11623,17 @@
...
+      "description": "iOS Client for NewsAPI.org",
+      "category-ids": ["news"],
+      "tags": ["swift"],
```
Comment (dkhamsing) [inline contents.json]: can u also add the tag "news api"

File: contents.json [hunk 3]
```diff
@@ -11623,6 +11623,17 @@
...
+      "screenshots": ["https://github.com/paulsumit1993/DailyFeed/blob/master/assets/screenshot.jpg"],
```
Comment (dkhamsing) [inline contents.json]: this should be https://raw.githubusercontent.com/paulsumit1993/DailyFeed/master/assets/screenshot.jpg

Comment (paulsumit1993) [inline contents.json]: my bad, will fix it

File: contents.json [hunk 4]
```diff
@@ -11623,6 +11623,17 @@
       "license": "mit",
       "date_added": "Jan 25 2017",
       "suggested_by": "@dkhamsing"
+    },
+    {
+      "title": "DailyFeed",
+      "source": "https://github.com/paulsumit1993/DailyFeedr",
+      "description": "iOS Client for NewsAPI.org",
```
Comment (dkhamsing) [inline contents.json]: pls change this to
Client for NewsAPI.org

iOS is assumed

**Comment by dkhamsing** [comment]
thx

**Pull request merged**